std-bits/table/src/main/java/org/enso/table/data/index/MultiValueKeyBase.java

```diff
@@ -31,10 +32,13 @@ protected Object get(int column) {
 
   /** Checks if all cells in the current row are missing. */
   public boolean areAllNull() {
+    Context context = Context.getCurrent();
     for (Storage<?> storage : storages) {
       if (!storage.isNa(rowIndex)) {
         return false;
       }
+
+      context.safepoint();
     }
     return true;
   }
```

Comment:
In [e194b62](https://github.com/enso-org/enso/pull/7183/commits/e194b62a8dbf50e88c1918ac87ed7913f3640c28) I'm adding safepoints also to some loops in MultiValueKeys that are bounded by the number of columns.

These are rather tight loops, so I'm not sure if adding the safepoints here is worth it. But technically we could have tables with 100s of columns where this could start being useful.

(There are other bottlenecks with tables with so many columns though, so such big tables will not work very smoothly without some other improvements anyway, but maybe it's worth including at least this)

I'm a bit on the fence if this particular commit is worth it - @JaroslavTulach @jdunkerley what do you think about this?